tools/myst/host/package.c

```diff
@@ -580,6 +582,10 @@ int _exec_package(
     int ret = -1;
     const char** exec_args = NULL;
     myst_args_t mount_mappings = {0};
+    myst_fssig_t fssig;
```

Comment:
The ext2 rootfs roothash is part of the rootfs image, the function should be able to extract the roothash info from the image instead of requiring the caller to provide it as input.

Reply:
Thinking over the difference between "packaged" mode and myst-exec mode, I started doubt the relevance of the config_id to the "packaged" mode. In the packaged mode, the app configuration data is part of the enclave image, the CPIO rootfs or the roothash of an EXT2 rootfs is also part of the enclave image, with the enclave built/signed in a trusted build process. The value of config_id is to enable  "loading" different app rootfs/configuration into a prebuilt enclave, which might be provided by and third party. 

With config_id, "myst exec" mode will be able to support "loading" different EXT2 rootfs/configuration into a pre-built enclave. In current implementation, "myst_exec-sgx" command build and sign the enclave, on the running platform (typically, not a trusted build process, therefore "myst exce-sgx" is suitable for dev/debug purpose only). With config_id, an enclave prebuilt in a trusted build process can be used to support `myst exec-sgx [ext2roorfs] [--app-config-path config.json]`